kvm-ioctls/CHANGELOG.md

```diff
@@ -29,6 +29,8 @@
 - [[#273](https://github.com/rust-vmm/kvm-ioctls/pull/273)]: `DeviceFd::get_device_attr` is now
   marked as unsafe.
 - [[#277](https://github.com/rust-vmm/kvm-ioctls/pull/277)]: Updated kvm-bindings to 0.9.1.
+- [[#288](https://github.com/rust-vmm/kvm-ioctls/pull/288)]: Add kvm guest memfd related
```

Comment:
The entry for this PR should be in the [unreleased] section (right now its in the 0.19.0 section, but that version was already released). It also needs to be updated to only state that `VmFd::check_extension_int` and `VmFd::check_extension_raw` were added (since the guest_memfd stuff already made it into 0.18.0)